### ISSUE TITLE
target.arm-unknown-linux-gnueabihf.image doesn't work

### ISSUE THREAD

**Issue by hombit** [cross-rs/cross#414]
Default image used by `cross` is broken for `arm-unknown-linux-gnueabihf`, `rustembedded/cross:arm-unknown-linux-gnueabihf-0.2.0` (see #391). But after I specified image fixed in #397 in my Caro.toml (see bellow), `cross` continued to use default image.

Command I run:
```sh
cross build --release --target=arm-unknown-linux-gnueabihf
```

`Cargo.toml` modification I made:
```toml
[target.arm-unknown-linux-gnueabihf]
image = "rustembedded/cross:arm-unknown-linux-gnueabihf"
```

Tested on both `0.2.0` and current `master` (5d717f08)

**Comment by Darneas** [comment]
You must put the two-line modification into `Cross.toml` not `Cargo.toml`.

**Comment by hombit** [comment]
@Darneas Thank you! But what about default image for `arm-unknown-linux-gnueabihf`?

**Comment by Darneas** [comment]
The default one with `0.2.0` seems to remain broken. I am unsure how the `cross` project handles these version numbers and synchronizes them across all targets.

So maybe one of the maintainers needs to look into this. Maybe @reitermarkus as he also did #392?

**Comment by BrianHoldsworth** [comment]
I believe the problem persists in the latest docker image for arm-unknown-linux-gnueabihf. I'm trying to target the Pi Zero W. I *did not* make the mistake pointed out above, and DID add the following in my simple Rust project's `Cross.toml` file:
```
[target.arm-unknown-linux-gnueabihf]
image = "rustembedded/cross:arm-unknown-linux-gnueabihf"
```
AND I can confirm using `docker images` after running `cross build` that I'm not using the older `arm-unknown-linux-gnueabihf-0.2.0`
```
REPOSITORY           TAG                           IMAGE ID            CREATED             SIZE
rustembedded/cross   arm-unknown-linux-gnueabihf   ba2dc7364b10        5 days ago          2.2GB
```
It appears to me that `cross build` is creating an ARMv7 binary, and this causes the Seqmentation Fault on the Pi Zero, which is an ARMv6 CPU.
```
$ !readelf
readelf --arch-specific target/arm-unknown-linux-gnueabihf/debug/rpi-test
Attribute Section: aeabi
File Attributes
  Tag_CPU_name: "7-A"
  Tag_CPU_arch: v7
  Tag_CPU_arch_profile: Application
  Tag_ARM_ISA_use: Yes
  Tag_THUMB_ISA_use: Thumb-2
  Tag_FP_arch: VFPv3-D16
  Tag_ABI_PCS_GOT_use: GOT-indirect
  Tag_ABI_PCS_wchar_t: 4
  Tag_ABI_FP_rounding: Needed
  Tag_ABI_FP_denormal: Needed
  Tag_ABI_FP_exceptions: Needed
  Tag_ABI_FP_number_model: IEEE 754
  Tag_ABI_align_needed: 8-byte
  Tag_ABI_enum_size: int
  Tag_ABI_VFP_args: VFP registers
  Tag_CPU_unaligned_access: v6
  Tag_ABI_FP_16bit_format: IEEE 754
```

**Comment by Alexhuszagh** [comment]
Should be fixed as of #734.
